discord-service/src/test/groovy/pl/flyingoctopus/trello/service/impl/TrelloServiceImplTest.groovy

```diff
@@ -0,0 +1,21 @@
+package pl.flyingoctopus.trello.service.impl
+
+import org.springframework.http.HttpStatus
+import pl.flyingoctopus.trello.configuration.TrelloProperties
+import spock.lang.Specification
+
+class TrelloServiceImplTest extends Specification {
+
+    TrelloProperties trelloProperties = Mock()
+    TrelloServiceImpl trelloService = new TrelloServiceImpl(trelloProperties)
+
+    def "inviteToWorkspace should return HttpStatus"() {
```

Comment:
Tutaj trochę słabo to wygląda, bo nie sprawdzasz żadnej interakcji z WebClientem. Przez to, że inicializacja jest w polu z buildera to nie da się tego ładnie zamockować.
W zasadzie w TrelloService zależy nam na tym, żeby chowało wysyłanie requestów REST. Więc testy powinny sprawdzać, czy takie requesty wysyła.

Warto tutaj w TrelloServiceImplTest, zamiast lombokowego @RequiredArgsConstructor zrobić swój i z dodatkowym parametrerm `WebClient.Builder`.
Tutaj w dokumentacji opisane jak wstrzykiwać WebClient https://docs.spring.io/spring-boot/docs/2.0.3.RELEASE/reference/html/boot-features-webclient.html

Wtedy możesz spokojnie zamockować WebClient i sprawdzić, czy jest odpowiednio wywoływany w serwisie.